resources/conf/config.toml

```diff
@@ -84,6 +84,20 @@
     localClaim = "CUSTOM-CLAIM"
 
 
+# Issuer 2
+[[enforcer.jwtTokenConfig]]
+    name="MGW"
+    issuer = "https://localhost:9095/token"
+    certificateAlias = "mgw"
+    # URL of the JWKs endpoint
+    jwksURL = ""
+    # Validate subscribed APIs
+    validateSubscription = false
+    # The claim in which the consumer key of the application is coming
+    consumerKeyClaim = "azp"
```

Comment:
Do we have this claim in the generated token?

Reply:
No, we don't. Will remove this from the 2nd Issuer.

Reply:
Kept this as empty.